lib/rom/setup.rb

```diff
@@ -54,8 +33,8 @@ def schema(&block)
     #   end
     #
     # @api public
-    def relation(name, &block)
-      @relations.update(name => block)
+    def relation(name, options = {}, &block)
+      @relations.update(name => [options, block])
```

Comment:
I know it's not the scope of this PR, but I can use a relation name twice and it won't blow up?